crates/node/builder/src/builder/mod.rs

```diff
@@ -162,6 +162,14 @@ impl<ChainSpec> NodeBuilder<(), ChainSpec> {
     pub const fn new(config: NodeConfig<ChainSpec>) -> Self {
         Self { config, database: () }
     }
+
+    /// Apply a function to the builder
+    pub fn apply<F>(self, f: F) -> Self
+    where
+        F: FnOnce(Self) -> Self,
+    {
+        f(self)
+    }
```

Comment:
exactly like this, but we also want this on 

https://github.com/paradigmxyz/reth/blob/43fe46f0d340fdf27e6b1b51916d0ecf9f6c314b/crates/node/builder/src/builder/mod.rs#L392-L392

Reply:
thank you for reviewing on the weekend. Updated!